app/controllers/ownership_calls_controller.rb

```diff
@@ -22,11 +22,21 @@ def create
   end
 
   def close
-    @ownership_call = @rubygem.ownership_call
-    if @ownership_call&.close
+    if @ownership_call.close
       redirect_to rubygem_path(@rubygem.slug), notice: t("ownership_calls.update.success_notice", gem: @rubygem.name)
     else
       redirect_to rubygem_adoptions_path(@rubygem.slug), alert: t("try_again")
     end
   end
+
+  private
+
+  def find_ownership_call
+    @ownership_call = @rubygem.ownership_call
+    if @ownership_call
```

Comment:
Pundit talks about a NilClassPolicy. What happens is that instead of the "try again" error, you get `render_forbidden` because the NilClassPolicy rejects the authorization.

This was the best I could come up with given existing code. I don't like that our response to "not found" is to say "try again", but I didn't want to get into a big refactor of a piece of code that no one uses.

Reply:
Is this preferred:

```ruby
  def find_ownership_call
    @ownership_call = @rubygem.ownership_call
    return authorize @ownership_call if @ownership_call
    redirect_to rubygem_adoptions_path(@rubygem.slug), alert: t("try_again")
  end
```